src/Items/ItemHandler.cpp

```diff
@@ -63,16 +63,15 @@
 
 
 
-bool cItemHandler::m_HandlerInitialized = false;
-cItemHandler * cItemHandler::m_ItemHandler[2268];
+std::array<std::vector<std::unique_ptr<cItemHandler>>, E_ITEM_LAST + 1> cItemHandler::m_ItemHandler;
```

Comment:
I thought std::array is fixed-sized and defined at compile time? Was its use intended?